public/apps/configuration/panels/audit-logging/constants.tsx

```diff
@@ -0,0 +1,185 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+export const CONFIG_LABELS = {
+  AUDIT_LOGGING: 'Audit logging',
+  GENERAL_SETTINGS: 'General settings',
+  LAYER_SETTINGS: 'Layer settings',
+  ATTRIBUTE_SETTINGS: 'Attribute settings',
+  IGNORE_SETTINGS: 'Ignore settings',
+  COMPLIANCE_SETTINGS: 'Compliance settings',
+  COMPLIANCE_CONFIG_SETTINGS: 'Config settings',
+  COMPLIANCE_READ: 'Read',
+  COMPLIANCE_WRITE: 'Write',
+};
+
+export const RESPONSE_MESSAGES = {
+  FETCH_ERROR_TITLE: 'Sorry, there was an error fetching audit configuration.',
+  FETCH_ERROR_MESSAGE:
+    'Please ensure hot reloading of audit configuration is enabled in the security plugin.',
+  UPDATE_SUCCESS: 'Audit configuration was successfully updated.',
+  UPDATE_FAILURE: 'Audit configuration could not be updated. Please check configuration.',
+};
+
+export interface SettingDescription {
```

Comment:
Minor: Description is a plain descriptive text. This one is more like content/displayConfig to me.

Reply:
I see. I will change to SettingContent before merging the PR.